web/src/lib/stores/assets-store.svelte.ts

```diff
@@ -972,7 +1013,7 @@ export class AssetStore {
       rowWidth: Math.floor(viewportWidth),
     };
   }
-  #updateGeometry(bucket: AssetBucket, invalidateHeight: boolean) {
+  #updateGeometry(bucket: AssetBucket, invalidateHeight: boolean, noDefer: boolean = false) {
```

Comment:
Could think about an `options: { invalidateHeight: boolean, noDefer: boolean }` object instead, since reading just `true, true` in the method call does not really convey what it does.